vector/src/main/java/im/vector/app/features/settings/devices/v2/overview/SessionOverviewViewModel.kt

```diff
@@ -74,11 +78,16 @@ class SessionOverviewViewModel @AssistedInject constructor(
     }
 
     init {
+        refreshPushers()
         observeSessionInfo(initialState.deviceId)
         observeCurrentSessionInfo()
         observePushers(initialState.deviceId)
     }
 
+    private fun refreshPushers() {
+        activeSessionHolder.getActiveSession().pushersService().refreshPushers()
```

Comment:
Maybe we should use `activeSessionHolder.getSafeActiveSession()` instead of all the `activeSessionHolder.getActiveSession()` to avoid any crash?